packages/fiori/src/TimelineItem.js

```diff
@@ -88,6 +88,29 @@ const metadata = {
 			defaultValue: "-1",
 			noAttribute: true,
 		},
+
+		/**
+		 * Select orientation.
+		 *
+		 * @type {String}
+		 * @defaultvalue "vertical"
+		 * @public
+		 */
+		axisOrientation: {
```

Comment:
This property should be of the new enum type as well. Change its docs as well(add available options & since tag)

Reply:
Done